### DISCUSSION TITLE
720p mp4 video selection no longer available.

### DISCUSSION THREAD

**Discussion by reuben-crowder** [FreeTubeApp/FreeTube#5333] [General]
When clicking on the "Download video" button, the "720p mp4 video" selection is no longer available.

**Comment by absidue** [comment]
YouTube stopped offering that stream, which is why it is missing. Unfortunately we can't do anything about it to make it come back, sorry.

Related issues on other projects: https://github.com/iv-org/invidious/issues/4759, https://github.com/yt-dlp/yt-dlp/issues/10231

As an alternative you can either download the 360p stream or download both an audio-only and a video-only stream and combine them into a single file with a tool like ffmpeg.
